File: templates/matlab/reliability_maintenance_before_after.m
function fig = reliability_maintenance_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 3320, 'reliability and maintenance: before-after slope', 'reliability and maintenance', 'before-after slope');
end

File: templates/python/reliability_maintenance_before_after.py
"""reliability_maintenance_before_after: 可靠性与维修前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='reliability and maintenance: before-after slope'):
    return make_template_figure('slope', seed=3320, title=title, domain='reliability and maintenance', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
